packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/send/dr-requests.ts

```diff
@@ -12,6 +16,7 @@ const context = "ihe-gateway-v2-dr-saml-client";
 export type DRSamlClientResponse = SamlClientResponse & {
   gateway: XCAGateway;
   outboundRequest: OutboundDocumentRetrievalReq;
+  contentType: string | undefined;
```

Comment:
nit: could make `contentType` optional so we don't need to set it as `undefined` like we're doing on line 76